.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+log/*
```

check `src/main/resources/log4j2.xml`
```suggestion
logs/*
```